frb_example/pure_dart/rust/src/api/misc_no_twin_example_a.rs

```diff
@@ -218,3 +218,15 @@ pub struct StructWithRustAutoOpaqueWithNonCloneData {
 impl StructWithRustAutoOpaqueWithNonCloneData {
     pub fn f(&self) {}
 }
+
+#[cfg(feature = "test")]
+#[allow(non_snake_case)] // There is a bug in spec_generator that requires camel case
```

Comment:
That looks weird... could you please elaborate a bit more?

Reply:
I noticed that in frb_generated.rs it sometimes swaps between camel case and snake case in ways that cause compilation failures. I can try to reproduce it again shortly.

Reply:
https://github.com/fmeef/flutter_rust_bridge/actions/runs/10938682186
this is an example of the issues that arise with camel case. Note the only change I made is switching `getFeatureTest()` to `get_feature_test()` and reran codegen. Correct me if I am wrong, but those should be equivalent in the generated code right?

```
error[E0425]: cannot find function `getFeatureTest` in module `crate::api::misc_no_twin_example_a`
    --> src/frb_generated.rs:7105:81
     |
7105 |                         Result::<_, ()>::Ok(crate::api::misc_no_twin_example_a::getFeatureTest())?;
     |                                                                                 ^^^^^^^^^^^^^^ help: a function with a similar name exists: `get_feature_test`
     |
    ::: src/api/misc_no_twin_example_a.rs:223:1
     |
223  | pub fn get_feature_test() -> String {
     | ----------------------------------- similarly named function `get_feature_test` defined here

For more information about this error, try `rustc --explain E0425`.
```

Reply:
That's weird... I have not seen anyone reporting their function has case problems...

In case it is issue with prefix `get_`, what about renaming the function (which also makes it more descriptive)

for example: `fn function_that_only_exists_under_feature_flag() { ... }`